admin/base/settings/defaults.py

```diff
@@ -233,9 +232,6 @@
     })
 
 
-ENTRY_POINTS = {'osf4m': 'osf4m', 'prereg_challenge_campaign': 'prereg',
```

Comment:
[this line was made obsolete when this line was removed](https://github.com/Johnetordoff/osf.io/commit/3a7e6ea06e76645182427ca94c5eddd4f02c71cc#diff-76a740757630125060c6bdefa75efb58L8)